Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoDecoderH264.mm

```diff
@@ -87,13 +117,17 @@ @implementation RTCVideoDecoderH264 {
   RTCVideoDecoderCallback _callback;
   OSStatus _error;
   bool _useAVC;
+  std::deque<std::unique_ptr<RTCVideoFrameWithOrder>> _reorderQueue;
```

Comment:
do WebRTC::lock work with `WTF_GUARDED_BY_LOCK(_reorderQueueLock)` it would be better to have that there.
Or at least a comment that it must be accessed with _reorderQueueLock held.

Personally, I like to put the lock definition above the variables it protects, in a paragraph-like style